packages/core/components/Icons/LoadingIcon.tsx

```diff
@@ -16,7 +17,8 @@ export default function LoadingIcon(props: Props) {
     return (
         <Spinner
             className={classNames(props.className, styles.spinner, {
-                [styles.invertColor]: props.invertColor,
+                [styles.invertColor]: props.invertColor && !props.highlightColor,
+                [styles.highlightColor]: props.highlightColor,
```

Comment:
maybe color should just be an enum at this point? seems odd highlightColor being true results in the same effect regardless of whether invertColor is true otherwise